examples/tutorials/data/hawc.py

```diff
@@ -80,21 +90,24 @@
 
 ######################################################################
 # Load the tables
+# ~~~~~~~~~~~~~~~
 
 data_path = "$GAMMAPY_DATA/hawc/crab_events_pass4/"
 hdu_filename = f"hdu-index-table-{energy_estimator}-Crab.fits.gz"
 obs_filename = f"obs-index-table-{energy_estimator}-Crab.fits.gz"
 
-# there is only one observation table
+######################################################################
+# There is only one observation table
 obs_table = ObservationTable.read(data_path + obs_filename)
 
-# we read the hdu index table of fHit bin number 6
+######################################################################
+# Now, we read the HDU index table of fHit bin number 6
```

Comment:
Maybe here specify that the remainder of the tutorial is made using only one single `fHit` value, but that a regular analysis would combine them all. (which is suggested in the exercices.